sublime-text-3/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+if [ -z ${NEWUSER+x} ]; then
+	echo 'WARN: No user was defined, defaulting to root.'
+	echo 'WARN: Sublime will save files as root:root.'
+	echo '      To prevent this, start the container with -e NEWUSER=$USER'
+	/usr/src/sublime_text/sublime_text -w
+else
+	# The root user already exists, so we only need to do something if 
+	# a user has been specified.
+	useradd -s /bin/bash $NEWUSER
+	su $NEWUSER -c "/usr/src/sublime_text/sublime_text -w -a /home/$NEWUSER/Documents"
```

Comment:
I don't have my code in "Documents".

```
- a /home/$NEWUSER/Documents 
```
forces sublime to open a file called "Documents" at runtime. 

Is this parameter really necessary?